src/core_plugins/kibana/public/dashboard/dashboard_config.js

```diff
@@ -0,0 +1,22 @@
+import { uiModules } from 'ui/modules';
+uiModules.get('kibana')
+  .provider('dashboardConfig', () => {
+    let hideWriteControls = false;
+
+    return {
+      /**
+       * Part of the exposed plugin API - do not remove without careful consideration.
+       * @type {boolean}
+       */
+      turnHideWriteControlsOn() {
+        hideWriteControls = true;
+      },
+      $get() {
+        return {
+          getHideWriteControls() {
```

Comment:
More of a general question, for getters that return booleans, should the getter be named `shouldHide...` or `canHide...`? It seems more expressive when calling it within conditionals.